src/users/users.njk

```diff
@@ -6,19 +6,14 @@
   Team members
 {% endblock %}
 
-{% block head %}
-  {{ super() }}
-  <link rel="stylesheet" type="text/css" href="./users.scss">
-{% endblock %}
-
 {% block main %}
 <div class="govuk-o-grid">
   <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
-    <h1 class="heading-large">
+    <h1 class="govuk-heading-l">
       Team members
```

Comment:
You took the `Team` away from the button. Do you think it's worth taking away from here too?

Reply:
I was leaving "team" here as I thought it was nice as a an introduction and we could shorten it to "members" everywhere else as people already had the context that these were "team members" from the heading.  Putting "team" in front of the heading also makes it seem a little bit less repetitive when the paragraph below starts with "members".

Reply:
Maybe we should have content designer look at it though.

Reply:
Maybe, my only worry is that introduction of a new unfamiliar to CF word which may confuse people. What is a team? We already have Organisation Members and Space Members etc.